### ISSUE TITLE
Cosine Distance Calculation

### ISSUE THREAD

**Issue by jiajingchen113322** [leaderj1001/CLIP#1]
Very appreciates the code here, it's very helpful to me.
However, in line 40 of the main.py, the logits you calculate might be the Cosine Similarity but not the Cosine Distance, should you put a minus Symbol there?
